### DISCUSSION TITLE
How to export/load custom VRMA?

### DISCUSSION THREAD

**Discussion by Benjythebee** [pixiv/three-vrm#1524] [Help]
To test importing VRMA animations, I have this [FBX boxing mixamo animation](https://github.com/Benjythebee/three-vrm-cache-repo/blob/master/Boxing.fbx).

I transformed it into a VRMA using [VRM Addon for blender (blender 4.2)](https://github.com/saturday06/VRM-Addon-for-Blender)
This is the output: https://github.com/Benjythebee/three-vrm-cache-repo/blob/master/cheering.vrma

Loading the VRMA output into [this three-vrm example](https://pixiv.github.io/three-vrm/packages/three-vrm-animation/examples/dnd.html) just doesn't work. 

Do you have a way to make advanced animations work as VRMA?

Please add tag "discussion" if possible

**Comment by Benjythebee** [comment]
For reference, I made an issue in [Saturday06's Blender Addon](https://github.com/saturday06/VRM-Addon-for-Blender/issues/633)  just in case

**Comment by 0b5vr** [comment]
@Benjythebee It seems that the y component of the hips position is approximately zero, which means that the animation is not T-posed appropriately.
I'm pretty sure you have to fix the animation side on Blender.

**Reply by Benjythebee** [reply]
The animation is straight from Mixamo though

**Reply by 0b5vr** [reply]
The conversion in the Blender side is not performed correctly I believe. Make sure your VRMA transforms follow the [VRM T-pose specification](https://github.com/vrm-c/vrm-specification/blob/master/specification/VRMC_vrm-1.0/tpose.md), [as the VRM Animation spec says](https://github.com/vrm-c/vrm-specification/tree/master/specification/VRMC_vrm_animation-1.0#humanoid).

**Comment by 0b5vr** [comment]
By the way, we intended to provide an appropriate warning when loading such files, but we failed to do so. We fixed the warning in https://github.com/pixiv/three-vrm/pull/1525 .
